.teamcity/_Self/Project.kt

```diff
@@ -27,4 +27,5 @@ object AdminAppProject : Project({
     buildType(_self.buildTypes.BuildPullRequests)
     buildType(_self.buildTypes.Deploy)
     buildType(_self.buildTypes.PublishPackage)
+    buildType(_self.buildTypes.PublishPackageToAzure)
```

Comment:
For greater clarity, can this be renamed to 
```suggestion
    buildType(_self.buildTypes.PublishPackageToAzureArtifacts)
```